OpenRA.Game/GameRules/WeaponInfo.cs

```diff
@@ -50,6 +50,9 @@ public sealed class WeaponInfo
 		[Desc("The sound played each time the weapon is fired.")]
 		public readonly string[] Report = null;
 
+		[Desc("Whether the weapon fires continuously instead of single shots.")]
```

Comment:
Isn't this just about the sound? The description suggests to me that this setting actually changes the firing behaviour of the weapon. What about `Whether the weapon fire sound plays continuously instead of single for each shot.`?